js/modules/databindings/chart.js

```diff
@@ -20,7 +20,12 @@ define(['knockout'], (ko) => {
   ko.bindingHandlers.chart = {
 		update: function (element, valueAccessor) {
       const chart = valueAccessor();
-			draw(chart.data(), element, chart.minHeight, chart.format, chart.renderer);
+      try {
+        draw(chart.data(), element, chart.minHeight, chart.format, chart.renderer);
+      } catch(er) {
+        console.error('Error when rendering chart');
```

Comment:
Why wouldn't you add exception object to logger?

Reply:
because first, it tells nothing (it doesn't know anything about chart's name), and second, the actual bug is in the `atlascharts` (it doesn't check data for validity)